### ISSUE TITLE
No updates to `self.logits` with bigram counts in Bigram class

### ISSUE THREAD

**Issue by mriganktiwari** [karpathy/makemore#28]
Hello everyone, i've made a `makemore.py` file (same as https://github.com/karpathy/makemore/blob/master/makemore.py) with only Bigram model class as of now.

while running it with `python makemore.py -i names.txt -o names --type bigram` arguments, wouldn't it just keep training, where it actually isn't doing anything coz there is no operation in forward pass.

Has anyone succesfully run it? Or it is yet to be completed?

**Comment by mriganktiwari** [comment]
i have made changes that make it work, it's quite hacky as is.
https://github.com/mriganktiwari/build_gpt/commit/e1140fda3fc3f5dac5563d86211b4e8486beb382
